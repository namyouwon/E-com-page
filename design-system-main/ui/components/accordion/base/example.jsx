// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import uniqueId from 'lodash.uniqueid';
import Accordion, { AccordionSection } from '../';
import { ActionOverflow } from '../../../components/menus/action-overflow/example';
import Card from '../../cards/';

export const AccordionExample = () => (
  <Accordion>
    <AccordionSection
      summary="Accordion summary"
      isOpen
      referenceId={uniqueId('referenceId-')}
    >
      Accordion details - A
    </AccordionSection>
    <AccordionSection
      summary="Accordion summary"
      referenceId={uniqueId('referenceId-')}
    >
      Accordion details - B
    </AccordionSection>
    <AccordionSection
      summary="Accordion summary"
      referenceId={uniqueId('referenceId-')}
    >
      Accordion details - C
    </AccordionSection>
  </Accordion>
);

export default <AccordionExample />;

export let states = [
  {
    id: 'section-one-open',
    label: 'Section one open',
    element: (
      <Accordion>
        <AccordionSection
          summary="Accordion summary"
          isOpen
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - A
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - B
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - C
        </AccordionSection>
      </Accordion>
    )
  },
  {
    id: 'section-two-open',
    label: 'Section two open',
    element: (
      <Accordion>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - A
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          isOpen
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - B
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - C
        </AccordionSection>
      </Accordion>
    )
  },
  {
    id: 'section-three-open',
    label: 'Section three open',
    element: (
      <Accordion>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - A
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - B
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          isOpen
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - C
        </AccordionSection>
      </Accordion>
    )
  },
  {
    id: 'multiple-sections-open',
    label: 'Multiple sections open',
    element: (
      <Accordion>
        <AccordionSection
          summary="Accordion summary"
          isOpen
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - A
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - B
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          isOpen
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - C
        </AccordionSection>
      </Accordion>
    )
  }
];

export let examples = [
  {
    id: 'long-heading-truncated',
    label: 'Long Section Heading with Truncation for larger screens',
    demoStyles: 'max-width: 500px;',
    storybookStyles: true,
    element: (
      <Card>
        <Accordion>
          <AccordionSection
            summary="Accordion summary Accordion summary Accordion summary Accordion summary Accordion summary Accordion summary Accordion summary Accordion summary Accordion summary Accordion summary"
            isOpen
            referenceId={uniqueId('referenceId-')}
          >
            Accordion details - A
          </AccordionSection>
          <AccordionSection
            summary="Accordion summary"
            referenceId={uniqueId('referenceId-')}
          >
            Accordion details - B
          </AccordionSection>
          <AccordionSection
            summary="Accordion summary"
            referenceId={uniqueId('referenceId-')}
          >
            Accordion details - C
          </AccordionSection>
        </Accordion>
      </Card>
    )
  },
  {
    id: 'nested-accordions',
    label: 'Nested Accordions',
    element: (
      <Accordion>
        <AccordionSection
          summary="Accordion summary"
          isOpen
          referenceId={uniqueId('referenceId-')}
        >
          <Accordion>
            <AccordionSection
              summary="Nested accordion summary"
              isOpen
              referenceId={uniqueId('referenceId-')}
            >
              Accordion details - A
            </AccordionSection>
            <AccordionSection
              summary="Nested accordion summary"
              referenceId={uniqueId('referenceId-')}
            >
              Accordion details - B
            </AccordionSection>
          </Accordion>
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - B
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - C
        </AccordionSection>
      </Accordion>
    )
  },
  {
    id: 'styled',
    label: 'Wrapped in Card',
    element: (
      <div className="slds-card">
        <Accordion>
          <AccordionSection
            summary="Accordion summary"
            isOpen
            referenceId={uniqueId('referenceId-')}
          >
            Accordion details - A
          </AccordionSection>
          <AccordionSection
            summary="Accordion summary"
            referenceId={uniqueId('referenceId-')}
          >
            Accordion details - B
          </AccordionSection>
          <AccordionSection
            summary="Accordion summary"
            referenceId={uniqueId('referenceId-')}
          >
            Accordion details - C
          </AccordionSection>
        </Accordion>
      </div>
    )
  },
  {
    id: 'has-action-overflow',
    label: 'Has Action Overflow Menu',
    element: (
      <Accordion>
        <AccordionSection
          summary="Accordion summary"
          actionMenu={
            <ActionOverflow position="right" dropdownIsOpen={false} />
          }
          isOpen
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - A
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          actionMenu={
            <ActionOverflow position="right" dropdownIsOpen={false} />
          }
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - B
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          actionMenu={
            <ActionOverflow position="right" dropdownIsOpen={false} />
          }
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - C
        </AccordionSection>
      </Accordion>
    )
  },
  {
    id: 'section-background-color',
    label: 'Accordion Section - Background Color',
    demoStyles: '--slds-c-accordion-summary-color-background: lightblue',
    storybookStyles: true,
    element: (
      <Accordion>
        <AccordionSection
          summary="Accordion summary"
          isOpen
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - A
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - B
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - C
        </AccordionSection>
      </Accordion>
    )
  },
  {
    id: 'section-background-color',
    label: 'Accordion Section - Text Color',
    demoStyles:
      '--slds-c-accordion-heading-text-color: green; --slds-c-accordion-heading-text-color-hover: red',
    storybookStyles: true,
    element: (
      <Accordion>
        <AccordionSection
          summary="Accordion summary"
          isOpen
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - A
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - B
        </AccordionSection>
        <AccordionSection
          summary="Accordion summary"
          referenceId={uniqueId('referenceId-')}
        >
          Accordion details - C
        </AccordionSection>
      </Accordion>
    )
  }
];
