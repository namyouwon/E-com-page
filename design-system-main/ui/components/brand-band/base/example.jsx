// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';

import BrandBand from '../';

export default <BrandBand size="medium" />;

export let examples = [
  {
    id: 'brand-band-small',
    label: 'Small',
    element: <BrandBand size="small" />
  },
  {
    id: 'brand-band-large',
    label: 'Large',
    element: <BrandBand size="large" />
  },
  {
    id: 'brand-band-no-image',
    label: 'No Image',
    element: <BrandBand size="none" />
  },
  {
    id: 'brand-band-large-group',
    label: 'Large Group',
    element: <BrandBand size="large" image="group" />
  },
  {
    id: 'brand-band-large-user',
    label: 'Large User',
    element: <BrandBand size="large" image="user" />
  }
];
