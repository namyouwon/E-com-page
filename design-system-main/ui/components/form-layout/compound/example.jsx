// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import PropTypes from 'prop-types';
import { FormElement, Fieldset } from '../../form-element/';
import Input from '../../input/';

export const CompoundForm = (props) => {
  const { hasTooltip, isRequired } = props;

  return (
    <React.Fragment>
      <Fieldset
        hasCompoundFields
        hasTooltip={hasTooltip}
        isRequired={isRequired}
        isDeprecated
        label="Location"
      >
        <div className="slds-form-element__group">
          <div className="slds-form-element__row">
            <FormElement
              formElementClassName="slds-size_1-of-2"
              labelContent="Latitude"
              inputId="input-10"
            >
              <Input id="input-10" />
            </FormElement>
            <FormElement
              formElementClassName="slds-size_1-of-2"
              labelContent="Longitude"
              inputId="input-11"
            >
              <Input id="input-11" />
            </FormElement>
          </div>
        </div>
      </Fieldset>
      <Fieldset
        hasCompoundFields
        label="Shipping Address"
        isAddress
        isDeprecated
      >
        <div className="slds-form-element__group">
          <div className="slds-form-element__row">
            <FormElement
              formElementClassName="slds-size_1-of-1"
              labelContent="Shipping Street"
              inputId="input-12"
            >
              <Input id="input-12" />
            </FormElement>
          </div>
          <div className="slds-form-element__row">
            <FormElement
              formElementClassName="slds-size_4-of-6"
              labelContent="Shipping City"
              inputId="input-13"
            >
              <Input id="input-13" />
            </FormElement>
            <FormElement
              formElementClassName="slds-size_2-of-6"
              labelContent="Shipping State/Province"
              inputId="input-14"
            >
              <Input id="input-14" defaultValue="MT" />
            </FormElement>
          </div>
          <div className="slds-form-element__row">
            <FormElement
              formElementClassName="slds-size_4-of-6"
              labelContent="Shipping Zip/Postal Code"
              inputId="input-15"
            >
              <Input id="input-15" />
            </FormElement>
            <FormElement
              formElementClassName="slds-size_2-of-6"
              labelContent="Shipping Country"
              inputId="input-16"
            >
              <Input id="input-16" defaultValue="Canada" />
            </FormElement>
          </div>
        </div>
      </Fieldset>
    </React.Fragment>
  );
};

CompoundForm.propTypes = {
  hasTooltip: PropTypes.bool,
  isRequired: PropTypes.bool,
};
