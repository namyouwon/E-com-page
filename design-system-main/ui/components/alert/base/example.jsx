// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import Alert from '../';
import { UtilityIcon } from '../../icons/base/example';
import EmptyLink from '../../../shared/empty-link/';
import Heading from '../../../shared/heading/';

export default [
  {
    id: 'default',
    label: 'Default',
    element: (
      <Alert>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="user"
        />
        <Heading level="2">
          Logged in as John Smith (johnsmith@acme.com).{' '}
          <EmptyLink>Log out</EmptyLink>
        </Heading>
      </Alert>
    )
  }
];

export let states = [
  {
    id: 'theme-info',
    label: 'Default (Theme Class  - Deprecated)',
    element: (
      <Alert isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="user"
        />
        <Heading level="2">
          Logged in as John Smith (johnsmith@acme.com).{' '}
          <EmptyLink>Log out</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'warning',
    label: 'Warning',
    element: (
      <Alert type="warning">
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="warning"
        />
        <Heading level="2">
          Your browser is outdated. Your Salesforce experience may be degraded.{' '}
          <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'theme-warning',
    label: 'Warning (Theme Class  - Deprecated)',
    element: (
      <Alert type="warning" isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="warning"
        />
        <Heading level="2">
          Your browser is outdated. Your Salesforce experience may be degraded.{' '}
          <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'error',
    label: 'Error',
    element: (
      <Alert type="error">
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="error"
        />
        <Heading level="2">
          Your browser is currently not supported. Your Salesforce may be
          degraded. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'theme-error',
    label: 'Error (Theme Class  - Deprecated)',
    element: (
      <Alert type="error" isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="error"
        />
        <Heading level="2">
          Your browser is currently not supported. Your Salesforce may be
          degraded. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'offline',
    label: 'Offline',
    element: (
      <Alert type="offline">
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="offline"
        />
        <Heading level="2">
          You are in offline mode. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'theme-offline',
    label: 'Offline (Theme Class  - Deprecated)',
    element: (
      <Alert type="offline" isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="offline"
        />
        <Heading level="2">
          You are in offline mode. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  }
];
