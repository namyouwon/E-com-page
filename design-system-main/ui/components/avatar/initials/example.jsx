// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import { Avatar } from '../base/example';
import { AvatarInitials } from '../';

export default [
  {
    id: 'default',
    label: 'Default',
    element: (
      <Avatar>
        <AvatarInitials
          iconClassName="slds-icon-standard-account"
          title="company name"
          initials="Ac"
        />
      </Avatar>
    )
  }
];

export let examples = [
  {
    id: 'linked-avatar-initials',
    label: 'Wrapped in a link',
    element: (
      <a href="#" onClick={e => e.preventDefault()}>
        <Avatar>
          <AvatarInitials
            iconClassName="slds-icon-standard-account"
            title="company name"
            initials="Ac"
          />
        </Avatar>
      </a>
    )
  }
];
