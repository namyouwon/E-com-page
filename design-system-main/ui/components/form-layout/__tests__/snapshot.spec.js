// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license
/* eslint-env jest */

import React from 'react';
import createHelpers from '../../../../jest.helpers';
import { Form } from '../base/example';
import { CompoundForm } from '../compound/example';

const { matchesMarkup } = createHelpers(__dirname);

describe('Form Layout', () => {
  it('renders a base form layout', () => matchesMarkup(<Form />));

  it('renders a base form layout stacked', () =>
    matchesMarkup(<Form isStacked />));

  it('renders a base form layout horizontally', () =>
    matchesMarkup(<Form isHorizontal />));

  it('renders a compound form layout', () => matchesMarkup(<CompoundForm />));

  it('renders a compound form layout with a tooltip', () =>
    matchesMarkup(<CompoundForm tooltip />));
});
