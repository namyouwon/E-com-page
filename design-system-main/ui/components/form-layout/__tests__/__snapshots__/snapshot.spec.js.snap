// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Layout renders a base form layout 1`] = `
<div
  className="slds-form"
>
  <div
    className="slds-form-element"
  >
    <label
      className="slds-form-element__label"
      htmlFor="input-id-01"
    >
      Text Input
    </label>
    <div
      className="slds-form-element__control"
    >
      <input
        className="slds-input"
        id="input-id-01"
        type="text"
      />
    </div>
  </div>
  <div
    className="slds-form-element"
  >
    <label
      className="slds-form-element__label"
      htmlFor="input-id-02"
    >
      Textarea Label
    </label>
    <div
      className="slds-form-element__control"
    >
      <textarea
        className="slds-textarea"
        id="input-id-02"
        placeholder="Placeholder text…"
      />
    </div>
  </div>
  <fieldset
    className="slds-form-element"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Checkbox Group Label
    </legend>
    <div
      className="slds-form-element__control"
    >
      <div
        className="slds-checkbox"
      >
        <input
          aria-labelledby={null}
          id="checkbox-unique-id-1"
          name="default"
          type="checkbox"
          value="checkbox-unique-id-1"
        />
        <label
          className="slds-checkbox__label"
          htmlFor="checkbox-unique-id-1"
        >
          <span
            className="slds-checkbox_faux"
          />
          <span
            className="slds-form-element__label"
          >
            All opportunities owned by you
          </span>
        </label>
      </div>
      <div
        className="slds-checkbox"
      >
        <input
          aria-labelledby={null}
          id="checkbox-unique-id-2"
          name="default"
          type="checkbox"
          value="checkbox-unique-id-2"
        />
        <label
          className="slds-checkbox__label"
          htmlFor="checkbox-unique-id-2"
        >
          <span
            className="slds-checkbox_faux"
          />
          <span
            className="slds-form-element__label"
          >
            All contacts in the account owned by you
          </span>
        </label>
      </div>
    </div>
  </fieldset>
  <fieldset
    className="slds-form-element"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Radio Group Label
    </legend>
    <div
      className="slds-form-element__control"
    >
      <span
        className="slds-radio"
      >
        <input
          aria-labelledby={null}
          id="radio-1"
          name="options"
          type="radio"
          value="radio-1"
        />
        <label
          className="slds-radio__label"
          htmlFor="radio-1"
        >
          <span
            className="slds-radio_faux"
          />
          <span
            className="slds-form-element__label"
          >
            Lead generation
          </span>
        </label>
      </span>
      <span
        className="slds-radio"
      >
        <input
          aria-labelledby={null}
          id="radio-2"
          name="options"
          type="radio"
          value="radio-2"
        />
        <label
          className="slds-radio__label"
          htmlFor="radio-2"
        >
          <span
            className="slds-radio_faux"
          />
          <span
            className="slds-form-element__label"
          >
            Education leads
          </span>
        </label>
      </span>
    </div>
  </fieldset>
</div>
`;

exports[`Form Layout renders a base form layout horizontally 1`] = `
<div
  className="slds-form slds-form_horizontal"
>
  <div
    className="slds-form-element"
  >
    <label
      className="slds-form-element__label"
      htmlFor="input-id-01"
    >
      Text Input
    </label>
    <div
      className="slds-form-element__control"
    >
      <input
        className="slds-input"
        id="input-id-01"
        type="text"
      />
    </div>
  </div>
  <div
    className="slds-form-element"
  >
    <label
      className="slds-form-element__label"
      htmlFor="input-id-02"
    >
      Textarea Label
    </label>
    <div
      className="slds-form-element__control"
    >
      <textarea
        className="slds-textarea"
        id="input-id-02"
        placeholder="Placeholder text…"
      />
    </div>
  </div>
  <fieldset
    className="slds-form-element"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Checkbox Group Label
    </legend>
    <div
      className="slds-form-element__control"
    >
      <div
        className="slds-checkbox"
      >
        <input
          aria-labelledby={null}
          id="checkbox-unique-id-5"
          name="default"
          type="checkbox"
          value="checkbox-unique-id-5"
        />
        <label
          className="slds-checkbox__label"
          htmlFor="checkbox-unique-id-5"
        >
          <span
            className="slds-checkbox_faux"
          />
          <span
            className="slds-form-element__label"
          >
            All opportunities owned by you
          </span>
        </label>
      </div>
      <div
        className="slds-checkbox"
      >
        <input
          aria-labelledby={null}
          id="checkbox-unique-id-6"
          name="default"
          type="checkbox"
          value="checkbox-unique-id-6"
        />
        <label
          className="slds-checkbox__label"
          htmlFor="checkbox-unique-id-6"
        >
          <span
            className="slds-checkbox_faux"
          />
          <span
            className="slds-form-element__label"
          >
            All contacts in the account owned by you
          </span>
        </label>
      </div>
    </div>
  </fieldset>
  <fieldset
    className="slds-form-element"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Radio Group Label
    </legend>
    <div
      className="slds-form-element__control"
    >
      <span
        className="slds-radio"
      >
        <input
          aria-labelledby={null}
          id="radio-5"
          name="options"
          type="radio"
          value="radio-5"
        />
        <label
          className="slds-radio__label"
          htmlFor="radio-5"
        >
          <span
            className="slds-radio_faux"
          />
          <span
            className="slds-form-element__label"
          >
            Lead generation
          </span>
        </label>
      </span>
      <span
        className="slds-radio"
      >
        <input
          aria-labelledby={null}
          id="radio-6"
          name="options"
          type="radio"
          value="radio-6"
        />
        <label
          className="slds-radio__label"
          htmlFor="radio-6"
        >
          <span
            className="slds-radio_faux"
          />
          <span
            className="slds-form-element__label"
          >
            Education leads
          </span>
        </label>
      </span>
    </div>
  </fieldset>
</div>
`;

exports[`Form Layout renders a base form layout stacked 1`] = `
<div
  className="slds-form slds-form_stacked"
>
  <div
    className="slds-form-element"
  >
    <label
      className="slds-form-element__label"
      htmlFor="input-id-01"
    >
      Text Input
    </label>
    <div
      className="slds-form-element__control"
    >
      <input
        className="slds-input"
        id="input-id-01"
        type="text"
      />
    </div>
  </div>
  <div
    className="slds-form-element"
  >
    <label
      className="slds-form-element__label"
      htmlFor="input-id-02"
    >
      Textarea Label
    </label>
    <div
      className="slds-form-element__control"
    >
      <textarea
        className="slds-textarea"
        id="input-id-02"
        placeholder="Placeholder text…"
      />
    </div>
  </div>
  <fieldset
    className="slds-form-element"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Checkbox Group Label
    </legend>
    <div
      className="slds-form-element__control"
    >
      <div
        className="slds-checkbox"
      >
        <input
          aria-labelledby={null}
          id="checkbox-unique-id-3"
          name="default"
          type="checkbox"
          value="checkbox-unique-id-3"
        />
        <label
          className="slds-checkbox__label"
          htmlFor="checkbox-unique-id-3"
        >
          <span
            className="slds-checkbox_faux"
          />
          <span
            className="slds-form-element__label"
          >
            All opportunities owned by you
          </span>
        </label>
      </div>
      <div
        className="slds-checkbox"
      >
        <input
          aria-labelledby={null}
          id="checkbox-unique-id-4"
          name="default"
          type="checkbox"
          value="checkbox-unique-id-4"
        />
        <label
          className="slds-checkbox__label"
          htmlFor="checkbox-unique-id-4"
        >
          <span
            className="slds-checkbox_faux"
          />
          <span
            className="slds-form-element__label"
          >
            All contacts in the account owned by you
          </span>
        </label>
      </div>
    </div>
  </fieldset>
  <fieldset
    className="slds-form-element"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Radio Group Label
    </legend>
    <div
      className="slds-form-element__control"
    >
      <span
        className="slds-radio"
      >
        <input
          aria-labelledby={null}
          id="radio-3"
          name="options"
          type="radio"
          value="radio-3"
        />
        <label
          className="slds-radio__label"
          htmlFor="radio-3"
        >
          <span
            className="slds-radio_faux"
          />
          <span
            className="slds-form-element__label"
          >
            Lead generation
          </span>
        </label>
      </span>
      <span
        className="slds-radio"
      >
        <input
          aria-labelledby={null}
          id="radio-4"
          name="options"
          type="radio"
          value="radio-4"
        />
        <label
          className="slds-radio__label"
          htmlFor="radio-4"
        >
          <span
            className="slds-radio_faux"
          />
          <span
            className="slds-form-element__label"
          >
            Education leads
          </span>
        </label>
      </span>
    </div>
  </fieldset>
</div>
`;

exports[`Form Layout renders a compound form layout 1`] = `
Array [
  <fieldset
    className="slds-form-element slds-form_compound"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Location
    </legend>
    <div
      className="slds-form-element__control"
    >
      <div
        className="slds-form-element__group"
      >
        <div
          className="slds-form-element__row"
        >
          <div
            className="slds-form-element slds-size_1-of-2"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-10"
            >
              Latitude
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-10"
                type="text"
              />
            </div>
          </div>
          <div
            className="slds-form-element slds-size_1-of-2"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-11"
            >
              Longitude
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-11"
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </fieldset>,
  <fieldset
    className="slds-form-element slds-form_compound slds-form-element_address"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Shipping Address
    </legend>
    <div
      className="slds-form-element__control"
    >
      <div
        className="slds-form-element__group"
      >
        <div
          className="slds-form-element__row"
        >
          <div
            className="slds-form-element slds-size_1-of-1"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-12"
            >
              Shipping Street
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-12"
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          className="slds-form-element__row"
        >
          <div
            className="slds-form-element slds-size_4-of-6"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-13"
            >
              Shipping City
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-13"
                type="text"
              />
            </div>
          </div>
          <div
            className="slds-form-element slds-size_2-of-6"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-14"
            >
              Shipping State/Province
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                defaultValue="MT"
                id="input-14"
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          className="slds-form-element__row"
        >
          <div
            className="slds-form-element slds-size_4-of-6"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-15"
            >
              Shipping Zip/Postal Code
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-15"
                type="text"
              />
            </div>
          </div>
          <div
            className="slds-form-element slds-size_2-of-6"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-16"
            >
              Shipping Country
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                defaultValue="Canada"
                id="input-16"
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </fieldset>,
]
`;

exports[`Form Layout renders a compound form layout with a tooltip 1`] = `
Array [
  <fieldset
    className="slds-form-element slds-form_compound"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Location
    </legend>
    <div
      className="slds-form-element__control"
    >
      <div
        className="slds-form-element__group"
      >
        <div
          className="slds-form-element__row"
        >
          <div
            className="slds-form-element slds-size_1-of-2"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-10"
            >
              Latitude
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-10"
                type="text"
              />
            </div>
          </div>
          <div
            className="slds-form-element slds-size_1-of-2"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-11"
            >
              Longitude
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-11"
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </fieldset>,
  <fieldset
    className="slds-form-element slds-form_compound slds-form-element_address"
  >
    <legend
      className="slds-form-element__legend slds-form-element__label"
    >
      Shipping Address
    </legend>
    <div
      className="slds-form-element__control"
    >
      <div
        className="slds-form-element__group"
      >
        <div
          className="slds-form-element__row"
        >
          <div
            className="slds-form-element slds-size_1-of-1"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-12"
            >
              Shipping Street
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-12"
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          className="slds-form-element__row"
        >
          <div
            className="slds-form-element slds-size_4-of-6"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-13"
            >
              Shipping City
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-13"
                type="text"
              />
            </div>
          </div>
          <div
            className="slds-form-element slds-size_2-of-6"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-14"
            >
              Shipping State/Province
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                defaultValue="MT"
                id="input-14"
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          className="slds-form-element__row"
        >
          <div
            className="slds-form-element slds-size_4-of-6"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-15"
            >
              Shipping Zip/Postal Code
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                id="input-15"
                type="text"
              />
            </div>
          </div>
          <div
            className="slds-form-element slds-size_2-of-6"
          >
            <label
              className="slds-form-element__label"
              htmlFor="input-16"
            >
              Shipping Country
            </label>
            <div
              className="slds-form-element__control"
            >
              <input
                className="slds-input"
                defaultValue="Canada"
                id="input-16"
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </fieldset>,
]
`;
